lib/bulbs-elements/util/prep-reading-list-analytics.js

```diff
@@ -0,0 +1,19 @@
+import prepGaWrapper from './prep-ga-wrapper';
+import getAnalyticsManager from './get-analytics-manager';
+
+export default function prepReadingListAnalytics (element, dimension12) {
+  let parent = element.closest('bulbs-reading-list-item')[0];
+
+  if (parent) {
+    this.parentAnalyticsDimensions = parent.dataset.contentAnalyticsDimensions;
+    this.href = parent.dataset.href;
+    this.title = parent.dataset.title;
+  }
+
+  let dimensionObject = {
+    dimension12: dimension12,
+  };
+
+  this.analyticsManager = getAnalyticsManager();
+  this.analyticsWrapper = prepGaWrapper(this.parentAnalyticsDimensions, dimensionObject);
```

Comment:
I think chris makes a good point about being clear in the class when properties are getting set.

@spra85 @brycedorn what would you think about about a pattern like this?

```js
Object.assign(this, makeReadingListAnalyticsProps(this, element, 'element'))
```

Otherwise the more explicit would be:

```js
const readingListProps = makeReadingListAnalyticsProps(this, element, 'element'))
this.foo = readingListProps.foo;
this.bar = readingListProps.bar;
```